Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM node:boron
 
 # Update and install packages
 RUN apt-get update && \
-    apt-get install -y python python-dev python-pip python-virtualenv zip rsync && rm -rf /var/lib/apt/lists/*
+    apt-get install -y python python-dev python-pip python-virtualenv zip rsync default-jdk && \
```

Comment:
@niranjan94 we using this build pack https://github.com/heroku/heroku-buildpack-jvm-common/blob/master/bin/java in HEROKU. it has mentioned as 1.8. I think this will help.